qgepqwat2ili/utils/qgep_export_utils.py

```diff
@@ -165,7 +165,7 @@ def base_common(self, row, type_name):
             "t_id": self.get_tid(row),
         }
 
-        if self.current_basket is not None:
+        if self.current_basket is not None and type_name != "organisation":
```

Comment:
@domi4484 Here you are not writing the current_basket if you have class organisation - is this the problem. The tests fail on organisation data.